bin/python-noenv.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-env -i /usr/bin/python "$@"
+env -i PBS_DEFAULT=$PBS_DEFAULT /usr/bin/python "$@"
```

Comment:
well, glad to know someone checked all possible tools one can run with `os.system` for possible issues